Element84/filmdrop-ui
# add darkmode config

**Related Issue(s):**

- NA

**Proposed Changes:**

1. Add basemap dark theme configuration variable
  - This was needed so that a custom basemap could be set & still override the style to dark mode while showing custom BASEMAP_HTML_ATTRIBUTION text value
2. Add logo max-height

BONUS:
3. Resolve security issue coming from babel

## To test

- make sure `BASEMAP_DARK_THEME` is not in config
- load app
- confirm basemap defaults to dark mode
- stop app
- add `BASEMAP_DARK_THEME: true` to config
- load app
- confirm basemap defaults to dark mode
- stop app
- add `BASEMAP_DARK_THEME: false` to config
- load app
- confirm basemap uses provider default style (should show colors)

**PR Checklist:**
